server/app/assets/javascripts/admin_entry_point.ts

```diff
@@ -33,6 +34,7 @@ import htmx from './htmx'
 
 window.addEventListener('load', () => {
   initializeEverything()
+  SessionTimeoutHandler.init()
```

Comment:
question (non-blocking): Is there a reason you're initializing this separately rather than including it in the `initializeEverything` method?

Reply:
I moved it into intializeEverything().